pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -0,0 +1,152 @@
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal, Nav, NavItem, TabContent, TabPane, TabContainer } from 'patternfly-react';
+
+import cockpit from "cockpit";
+import './listing.less';
+import 'cockpit-components-modifications.css';
+
+const _ = cockpit.gettext;
+
+/* Dialog for showing scripts to modify system
+ *
+ * Enables showing shell and ansible script. Shell one is mandatory and ansible one can be omitted.
+ *
+ */
+class ModificationsExportDialog extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            key: 1
+        };
+
+        this.handleSelect = this.handleSelect.bind(this);
+    }
+
+    handleSelect(key) {
+        this.setState({ key });
+    }
+
+    render() {
+        return (
+            <Modal show={this.props.show} className="automation-script-modal">
+                <Modal.Header>
+                    <Modal.Title>{ _("Automation script") }</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <TabContainer id="basic-tabs-pf" defaultActiveKey={1}>
+                        <React.Fragment>
+                            <Nav bsClass="nav nav-tabs nav-tabs-pf" onSelect={this.handleSelect}>
+                                <NavItem eventKey={1}>
+                                    {_("Shell script")}
+                                </NavItem>
+                                {this.props.ansible &&
+                                    <NavItem eventKey={2}>
+                                        {_("Ansible script")}
+                                    </NavItem>
+                                }
+                            </Nav>
+                            <TabContent animation>
+                                <TabPane eventKey={1}>
+                                    <pre>
+                                        {this.props.shell}
+                                    </pre>
+                                </TabPane>
+                                {this.props.ansible &&
+                                    <TabPane eventKey={2}>
+                                        {this.props.ansible}
+                                    </TabPane>
+                                }
+                            </TabContent>
+                        </React.Fragment>
+                    </TabContainer>
+                </Modal.Body>
+                <Modal.Footer>
+                    <Button bsStyle='default' className='btn-cancel' onClick={this.props.onClose}>
+                        { _("Cancel") }
```

Comment:
We're not really cancelling any action here, so _Close_ is probably a better verb.